Doppler.PushContact/Controllers/MessageController.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Doppler.PushContact.DopplerSecurity;
+using Doppler.PushContact.Services;
+using Doppler.PushContact.Services.Messages;
+using Doppler.PushContact.Models;
+
+namespace Doppler.PushContact.Controllers
+{
+    [Authorize(Policies.ONLY_SUPERUSER)]
+    [ApiController]
+    public class MessageController : ControllerBase
+    {
+        private readonly IMessageSender _messageSender;
+        private readonly IMessageRepository _messageRepository;
+        private readonly IPushContactService _pushContactService;
+
+        public MessageController(IPushContactService pushContactService, IMessageRepository messageRepository, IMessageSender messageSender)
+        {
+            _pushContactService = pushContactService;
+            _messageRepository = messageRepository;
+            _messageSender = messageSender;
+        }
+
+        [HttpPost]
+        [Route("message/{messageId}")]
+        public async Task<IActionResult> MessageByVisitorGuid([FromRoute] Guid messageId, [FromBody] string visitorGuid)
+        {
+            if (string.IsNullOrEmpty(visitorGuid) || string.IsNullOrWhiteSpace(visitorGuid))
+            {
+                return BadRequest($"'{nameof(visitorGuid)}' cannot be null, empty or whitespace.");
+            }
+            if (string.IsNullOrEmpty(messageId.ToString()) || string.IsNullOrWhiteSpace(messageId.ToString()))
+            {
+                return BadRequest($"'{nameof(messageId)}' cannot be null, empty or whitespace.");
+            }
```

Comment:
`IsNullOrWhiteSpace` is enought. 

```suggestion
            if (string.IsNullOrWhiteSpace(visitorGuid))
            {
                return BadRequest($"'{nameof(visitorGuid)}' cannot be null, empty or whitespace.");
            }
            if (string.IsNullOrWhiteSpace(messageId.ToString()))
            {
                return BadRequest($"'{nameof(messageId)}' cannot be null, empty or whitespace.");
            }
```

Reply:
It would be nice to do these kinds of validations during model binding with annotations. I think that it is out of the box with the framework.

But, it is not so important right now.